docs/reference/modules/node.asciidoc

```diff
@@ -41,6 +41,12 @@ A tribe node, configured via the `tribe.*` settings, is a special type  of
 coordinating only node that can connect to multiple clusters and perform
 search and other operations across all connected clusters.
 
```

Comment:
Can we also reword the intro sentence? Instead of "Besides that, each node serves one or more
 purpose" maybe "In addition, each node functions as one or more of the following node types:"